test/e2e/testnet.go

```diff
@@ -18,6 +18,7 @@ type Testnet struct {
 	nodes           []*Node
 	genesisAccounts []*Account
 	keygen          *keyGenerator
+	txSimNode       *TxSim
```

Comment:
Do we want just a single tx sim node? Would that be sufficient to load the testnet?

Maybe it's fine to start with one and not prematurely over-engineer it

Reply:
>Maybe it's fine to start with one and not prematurely over-engineer it

Indeed, I intended to do it incrementally, first by testing out a single instance of txsim and debugging it, then scaling up if the desired load is not met.